### PR TITLE
Add Istio slack channel to contact spots

### PR DESCRIPTION
**Pull request by jshaughn**
Fixes https://github.com/kiali/kiali/issues/3834

### CODE REVIEW

File: content/documentation/v1.32/faq/general/how-do-i-access-kiai.adoc
Comment (abonas) [inline content/documentation/v1.32/faq/general/how-do-i-access-kiai.adoc:8]: does this belong to the slack channel fix?

Comment (jshaughn) [reply content/documentation/v1.32/faq/general/how-do-i-access-kiai.adoc:8]: If all else if fine, please approve and will merge.

**Comment by abonas** [comment]
are we squash n merge or other merge policy these days?

**Comment by jshaughn** [comment]
@abonas In general we squash, we don't want to litter git history with incremental commits.  Also, it provides a chance to clean up the final commit comment.  We do very rarely merge, but only when there is some compelling reason to maintain the individual commits.  As a reviewer you only need to approve/comment/reject, the PR owner typically does the squash and merge unless it is otherwise arranged between the reviewer and author.  So, still waiting on your approval here :-)

**Review by abonas — APPROVED** [review]

**Pull request merged**
